addons/mail/static/src/owl/components/chatter/chatter_component_wrapper.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('mail.component.ChatterComponentWrapper', function (require) {
+'use strict';
+
+const { ComponentWrapper } = require('web.OwlCompatibility');
+const { tags: { xml } } = owl;
```

Comment:
I think it's because I copy/paste some other line where I needed also Component. You're right, it's not best way to do the import here.

```javascript
const { Component, tags: { xml } } = owl;
```